apps/backoffice-v2/src/pages/TransactionMonitoringAlerts/components/AlertsTable/columns.tsx

```diff
@@ -44,6 +44,10 @@ export const columns = [
     },
     header: 'Date & Time',
   }),
+  columnHelper.accessor('label', {
+    cell: info => <Badge variant={'secondary'}>{info.getValue()}</Badge>,
```

Comment:
Consider if `valueOrNA` or the component wrapper is needed here. Can label be falsy? And how does it look like when invalid data or no data returns from the API.

Reply:
`label` is always a string